src/app/icd/ICDManager.cpp

```diff
@@ -68,24 +76,23 @@ void ICDManager::UpdateIcdMode()
 
     ICDMode tempMode = ICDMode::SIT;
 
-    // TODO ICD LIT FIX DEPENDENCY ISSUE with app/util/IcdMonitoringTable.h and app/server:server
     // The Check In Protocol Feature is required and the slow polling interval shall also be greater than 15 seconds
     // to run an ICD in LIT mode.
-    // if (kSlowPollingInterval > kICDSitModePollingThreashold && SupportsCheckInProtocol())
-    // {
-    //     // We can only get to LIT Mode, if at least one client is registered to the ICD device
-    //     const auto & fabricTable = Server::GetInstance().GetFabricTable();
-    //     for (const auto & fabricInfo : fabricTable)
-    //     {
-    //         PersistentStorageDelegate & storage = Server::GetInstance().GetPersistentStorage();
-    //         IcdMonitoringTable table(storage, fabricInfo.GetFabricIndex(), 1);
-    //         if (!table.IsEmpty())
-    //         {
-    //             tempMode = ICDMode::LIT;
-    //             break;
-    //         }
-    //     }
-    // }
+    if (GetSlowPollingInterval() > GetSitModePollingThreshold() && SupportsCheckInProtocol())
```

Comment:
GetSITModePollingThreshold(), please.  Acronyms should probably stay all-caps in names.  That's what we do elsewere, including below in "ICDMode::SIT" and whatnot.

Reply:
changed to all caps for acronyms in the few instances here. I'll do another PR to change the name of `IcdTableMonitoring.cpp/h` to `ICDTableMonitoring` and any other case I see